complement/pipeline.yml

```diff
@@ -37,3 +40,60 @@ steps:
       automatic:
         - exit_status: 128
           limit: 3
+
+  - command:
+      # Download the latest checkout of Synapse develop
+      - "wget -N https://github.com/matrix-org/synapse/archive/develop.tar.gz -O synapse.tar.gz && tar -xzf synapse.tar.gz && cd synapse"
```

Comment:
Please use a stable release e.g `master`. We aren't interested in bleeding edge Synapse stuff here, we care about Complement instead.

Reply:
Good point. We'll be testing `develop` in Synapse's pipeline anyways.

Heads up that building Complement's [Dendrite.Dockerfile](https://github.com/matrix-org/complement/blob/master/dockerfiles/Dendrite.Dockerfile) will build Dendrite master however. We may want to change that to download the latest release tag instead?

Reply:
Hmm, probably yeah.